source/common/config/utility.h

```diff
@@ -36,6 +36,20 @@ class ApiTypeValues {
   const std::string Grpc{"GRPC"};
 };
 
+/**
+ * RateLimitSettings for discovery requests.
+ */
+struct RateLimitSettings {
+  // Default Max Tokens.
+  static const uint32_t DEFAULT_MAX_TOKENS = 100;
```

Comment:
Maybe switch to `DefaultMaxTokens` etc, I think this is the preferred caps these days.